### ISSUE TITLE
Accessibility needs to be addressed

### ISSUE THREAD

**Issue by thebe14** [futo-org/grayjay-android#292]
Let me start by saying wonderful idea, I like the app. The player is already better than the YT stock app. It autorotates, something is missing on each play that no longer aggravates me (to quote a famous person who shall remain unnamed), it plays in the background, etc. But. You tried to cram sooo much info on the screen that you could only get away it with it by resorting to tiny fonts. Not good, I cannot read it. You need to rethink the UI, take into consideration Android's accessibility settings. Otherwise this will be very hard to use by people that are no longer 20 years old. Thanks for the hard work!

**Comment by zoqin** [comment]
If I can add experience to support this
I know someone hard of sight she use big font on every app, Gray jay do not scale like other app
The ui is propably not dynamic and scalable wich is a big issue
Even tho she trully hate the original YouTube app she still use it as she can't use gray jay at all because of this issue

**Comment by dlehman83** [comment]
Allow me to add to the request.There are just a few changes that would make this app accessible.  
1. Respect android font settings.  
2. Either change the layout, or make it a configurable option.    The side by side layout leaves less room for the text.  I recommend the title and other info be under the thumbnail.  
3. Better indication of when a video has been watched.  There appears to be a little blue bar under the thumbnail, but it isn’t very clear. The layout change may help that.  

Thank you

**Comment by defiling9046** [comment]
@Zvonimir-FUTO this is kind of similar to #1018 as I believe moving to Material 3 would help mitigate this issue and I think that would be the best way forward although I understand it would be a big change.

**Comment by defiling9046** [comment]
@Zvonimir-FUTO after thinking about it for some more time do you think this should be the main accessibility tracker similar to the plugin request tracker and merge a lot of accessibility issues (including stuff like the material 3 feature request) into this so it is easier to manage?

That would make more sense rather than my previous suggestion of merging this into the material 3 issue.

**Comment by Zvonimir-FUTO** [comment]
Think these can be all collapsed into few things like make it easy to make UI bigger, make sure there is visible contrast in UI and add content labeling.

Don't think we need an issue tracker for those as it would be best to do it all in one go throughout the app during UI redesign.

**Comment by defiling9046** [comment]
> Don't think we need an issue tracker for those as it would be best to do it all in one go throughout the app during UI redesign.

@Zvonimir-FUTO is the Grayjay team discussing a UI redesign internally then if an issue tracker isn't needed?

It's hard to know what the plans for Grayjay are as I never see it being discussed in the Zulip unless I'm missing something.

Thx for replying.

**Comment by Zvonimir-FUTO** [comment]
Yes, it's a common topic and something we plan on doing "one day". However, due to time constraints and project being designed in a way where platforms can break our functionality at any time and we have to drop everything and work on a fix, it is really hard to plan and work on those bigger changes.

**Comment by Giesa-lab** [comment]
Yes, it's a longstanding gripe, but Android features has moved on with a workaround for now, and is fully explained in greater detail, with screenshots, at my original feature request... Hope it works for y'all folks 😊 I'd still like the two finger gesture zoom feature of Android to be requested by Grayjay, so that Android UI can do it's usual zoom thing, like in other Apps. 🙏🏼

**Comment by Giesa-lab** [comment]
*** Attn Developers *** 

This rather ancient article at Stack Overflow may shed some light on coding perhaps?

https://stackoverflow.com/questions/5216658/pinch-zoom-for-custom-view

Good luck with that 🍀 👍🏼
